### ISSUE TITLE
[Feature]: Control: allow matching destination aspect ratio for overrides without an input image

### ISSUE THREAD

**Issue by lbeltrame** [vladmandic/sdnext#3395]
### Feature description

Currently the code works when an input image is set. However, when doing text to image, there's no input image set, only overrides (unless the first image is used as input image), and thus the control images will get stretched to match the destination resolution. 

In case no input image is set, resizing should be done matching on the destination resolution (this is what the old ControlNet extension did).

**Comment by vladmandic** [comment]
i understand the problem, but not sure how to address it.

if there is input image, any unit can use that information to resize override image (that was recently added).
but its not the other way around - core loop cannot resize itself based on unit output.
also note that there can be any number of units, each having different override and working at different resolutions - what should core do then? resize to first unit and ignore the rest?

keeping this open for the moment.

**Comment by lbeltrame** [comment]
Good point. I think the original CN extension was kind of messy in that regard (I have no idea how it did things internally) because you could set it differently for any unit (and that generated eldritch abominations in case the various units did not match). 
Note that this AFAICS only occurs if you don't preprocess (I didn't test, though).

**Comment by vladmandic** [comment]
added, only took just over a year :)

<img width="569" height="194" alt="Image" src="https://github.com/user-attachments/assets/97dc2c01-28e5-4eab-9e9a-b15256ab0481" />
